src/styles/index.scss

```diff
@@ -118,14 +118,16 @@ h6 {
 }
 
 html,
-body,
-div,
 p,
 span,
 li,
 button {
   transition: background-color 100ms ease-in, background 100ms ease-in;
 }
 
+body, .top-nav, div {
+  transition: background-color 750ms;
```

Comment:
**suggestion** @wraeth-eth we could also add `color` here to smooth it out for text i think

```suggestion
  transition: background-color color 750ms;
```

Reply:
Something weird happens with the text when you add it here

https://user-images.githubusercontent.com/104132113/168051511-5f813461-6ade-42d7-abba-69279123400f.mov